sympy/core/tests/test_relational.py

```diff
@@ -662,6 +662,17 @@ def test_issue_10304():
     e = 1 + d*I
     assert simplify(Eq(e, 0)) is S.false
 
+
+def test_issue_10401():
```

Comment:
need to add this test case for ```Eq```.
```
x = symbols('x', finite=False)
Eq((x - 1), x)
```
at``` x = oo```, ```(x-1) == x``` but currently it's return ```Eq(x - 1, x)``` so need to add condition for this in```class Equality```